src/core/core-adapter.js

```diff
@@ -125,6 +127,24 @@ export default class Core {
     this.onVerticalSearch = config.onVerticalSearch || function () {};
   }
 
+  /**
+   * Initializes the {@link CoreAdapter} by providing it with an instance of the Core library.
+   */
+  init () {
+    const params = { apiKey: this._apiKey, experienceKey: this._experienceKey };
```

Comment:
Do we want to add the locale and possibly the configurationLabel ( 'STAGING' | 'PRODUCTION' ) here as well?

Reply:
I was going off what is in master. In master, it looks like `provideCore` only takes in `apiKey` and `experienceKey`. Is that the incorrect branch of `answers-core`?

Reply:
oh, I was thinking this was pointed at feature/search-service. This is code is correct for master, and I think it's fine to leave this pointed at master for now